setup.py

```diff
@@ -1,19 +1,22 @@
 from io import open
 from setuptools import setup
 
+with open('README.md') as read_me:
+    long_description = read_me.read()
+
 setup(
     name='Eel',
     version='0.11.0',
     author='Chris Knott',
     author_email='chrisknott@hotmail.co.uk',
+    url='https://github.com/samuelhwilliams/Eel',
     packages=['eel'],
     package_data={
         'eel': ['eel.js'],
     },
     install_requires=['bottle', 'bottle-websocket', 'future', 'whichcraft'],
     python_requires='>=2.6',
     description='For little HTML GUI applications, with easy Python/JS interop',
-    long_description=open('README.md', encoding='utf-8').readlines()[2],
+    long_description=long_description,
```

Comment:
```suggestion
    long_description=long_description,
    long_description_content_type='text/markdown',
```

We need to also explicitly include the content type for the long description otherwise PyPi will try to render it as reStructuredText, fail, and reject it.